src/com/mani/lang/Modules/system/system.java

```diff
@@ -68,6 +68,18 @@ public Object call(Interpreter interpreter, List<Object> arguments) {
                 return Mani.hadRuntimeError;
             }
         });
+
+        interpreter.addSTD("latestError", new ManiCallable() {
```

Comment:
Adds a new function to the "system" API to get the latest error message thrown.

Reply:
Awesome, i like the idea.